### PR TITLE
Add meeting minutes

### PR DESCRIPTION
**Pull request by AA666666**

### CODE REVIEW

**Comment by AA666666** [comment]
update group meeting minutes

**Comment by AA666666** [comment]
The final change is done. Please review it when you have time. Thanks.

**Review by Jaredhw — APPROVED** [review]

File: Documentation/Meeting Notes/Group Meeting Minutes.md [hunk 1]
```diff
@@ -68,3 +68,29 @@ Discussing the gitstats strategy
...
+issues created:\
+Add linting to the Continuous integration setup\
+Create the MemberList component which gets and displays users in a table from /get-users on the server\
+Research and document the approach to creating pages in React\
+Create the UserSearch.js file in client/src/ \
+Create an Express Get route called /get-users in server.js\
+Create the getUsers function in server.js\
+Standardize the local database setup process\
+Add 5 example rows to the User table\
+Create the User table in the database
```
Comment (JadenBalogh) [inline Documentation/Meeting Notes/Group Meeting Minutes.md:91]: All this information is available from just looking at the issues page, and should not be part of the meeting minutes.

File: Documentation/Meeting Notes/Group Meeting Minutes.md [hunk 2]
```diff
@@ -68,3 +68,29 @@ Discussing the gitstats strategy
...
  - All the issues are updated on the github for more info
 
+
+
+## November 3rd
+###### 1:30-2:00pm: 
+Reviewing each other's work
+
+###### 2:00-2:45pm:
+Discussing the plan for next week
```
Comment (JadenBalogh) [inline Documentation/Meeting Notes/Group Meeting Minutes.md:78]: When reading a meeting minutes file, the person reading it should have a decent understanding of the results of the meeting without having attended. For example, "we decided that each member should coordinate their meeting times for the week at the end of the meeting." Please take this feedback (and the feedback from last week) into account when writing these minutes. If you need help with understanding the expectations for the meeting minutes, please let us know.

**Review by JadenBalogh — APPROVED** [review]
Approved for this week. However, if the quality is not improved for next week's meeting, then changes will be required. Please review the feedback from this week and last week about the minutes document, and let us know about any questions or concerns.

**Pull request merged**